docs/cmdline-opts/write-out.md

```diff
@@ -250,6 +250,11 @@ the result.
 ## `time_total`
 The total time, in seconds, that the full operation lasted.
 
+## `tls_earlydata`
+The amount of bytes that were sent as TLSv1.3 early data. This is 0
+if this TLS feature was not used and negative if the data sent had
+been rejected by the server. (Added in 8.12.0)
```

Comment:
I think you should give a mention that it is only if [--tls-earlydata](https://curl.se/docs/manpage.html#--tls-earlydata) is used. Also I'm wondering what is the use case for this, is it something that you see as being utilized by users of the curl tool?

Reply:
I think it is of interest to curl users that want to see if their attempt to use early data was successful or not. We ourselves will make use of that in our test cases.

I can add cross mentions with `--tls-earlydata`. That's probably helpful.

Reply:
done.